docker-compose.yml

```diff
@@ -67,9 +67,9 @@ services:
       - -Dtracingstore.fossildb.address=fossildb
       - -Ddatastore.oxalis.uri=webknossos:9000
     links:
-      - fossildb
+      - fossildb-persisted:fossildb
```

Comment:
what’s the difference here? could this have been intentional?

Reply:
This persists the database changes, like for the `webknossos` service. Might be intential, but I don't think so. I guess you might know this^^ https://github.com/scalableminds/webknossos/pull/3428